static/css/external-site.less

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   cfgov-refresh
+   Third-Party links pages at /external-site/.
+   ========================================================================== */
+
+.external-site_container {
+    .content_bar {
+      border-top: 1px solid @gray-50;
```

Comment:
If not, use height and change the background instead of the border. Suggestion was only to stay consistent with the usage.